apps/food_select.py

```diff
@@ -203,7 +216,28 @@ def info_gather(self, names, meals, df_saved, food_dict, receipt_name):
                 f"How many? (Left to claim: {left_to_claim})", step=1.0, max_value=left_to_claim, min_value=0.0
             ),2)
 
-        return name, order, amount
+        return order, amount
+
+    def info_gather_remove(self, meals, df_saved, receipt_name):
+        """
+        Gathers info from user regarding removal claims
+        """
+        colo, cola = st.columns(2)
+        with colo:
+            order = st.selectbox("Select an order", options=meals, format_func=self.food_formatter)
+        receipt_df = df_saved[(df_saved['label'] == receipt_name)]
+        receipt_grpd = receipt_df.groupby('food').sum()
+        try:
+            amt_order_recorded = receipt_grpd.loc[order,'amount']
+        except:
+            amt_order_recorded = 0.0
+        
+        with cola:
+            amount = round(st.number_input(
+                f"How many? (Left to remove: {amt_order_recorded})", step=1.0, max_value=amt_order_recorded, min_value=0.0
+            ),2)
+        
+        return order, -amount
```

Comment:
Can we merge this function with the previous, maybe with a true/false or other variable?

Reply:
I have merged the common functionality for the first column, and added two smaller functions for the add/remove claim input that are called based on mode.